### PR TITLE
Fix [Accessibility]Inspect or AccessibilityInsights tool's rectangle can't focus on the entire FlowLayoutPanel/Panel/SplitContainer/TableLayoutPanel/TabControl/TextBox with scrollBar

### LINKED ISSUE #10540: [Accessibility]Inspect or AccessibilityInsights tool's rectangle can't focus on the entire FlowLayoutPanel/Panel/SplitContainer/TableLayoutPanel/TabControl/TextBox with scrollBar
### .NET version

9.0.0-alpha.1.23621.3

### Did it work in .NET Framework?

Yes

### Did it work in any of the earlier releases of .NET Core or .NET 5+?

1. Controls repro on .NET 9.0/8.0/7.0, not repro on 6.0 and previous version: FlowLayoutPanel, Panel, SplitContainer, TableLayoutPanel
2. Controls repro on .NET 9.0/8.0/7.0/6.0, not repro on 5.0 and previous version: TabControl
3. Controls repro on .NET 9.0/8.0, not repro on .NET 7.0 and previous version: TextBox

### Issue description

[Accessibility]Inspect or AccessibilityInsights tool's rectangle can't focus on the entire FlowLayoutPanel/Panel/SplitContainer/TableLayoutPanel/TabControl/TextBox with scrollBar

### Steps to reproduce

1. Open and run the attached reproduce application
[ScrollBarTest.zip](https://github.com/dotnet/winforms/files/13782718/ScrollBarTest.zip)
2. Launch Inspect or Accessibility Insight tool
3. Focus on the entire cotnrol

**Actual result:**
1. Controls repro on .NET 9.0/8.0/7.0, not repro on 6.0 and previous version: FlowLayoutPanel, Panel, SplitContainer, TableLayoutPanel
image
2. Controls repro on .NET 9.0/8.0/7.0/6.0, not repro on 5.0 and previous version: TabControl
image
3. Controls repro on .NET 9.0/8.0, not repro on .NET 7.0 and previous version: TextBox
image

**Expected result:**
1. Controls repro on .NET 9.0/8.0/7.0, not repro on 6.0 and previous version: FlowLayoutPanel, Panel, SplitContainer, TableLayoutPanel
image
2. Controls repro on .NET 9.0/8.0/7.0/6.0, not repro on 5.0 and previous version: TabControl
image
3. Controls repro on .NET 9.0/8.0, not repro on .NET 7.0 and previous version: TextBox
image

### PR DESCRIPTION
**Pull request by Epica3055**
Fixes #10540

## Proposed changes

- 
-  add Scrollable control to test project.
-  fix get rectangle logic same as [this pr](https://github.com/dotnet/winforms/pull/10532)
- 

## Regression? 

- Yes 

## Risk

- minimal 

### Before

image
image
image

### After

image
image
image

## Test methodology 

- 
- manual 
- 

 

## Test environment(s) 

- 9.0.100-alpha.1.23618.3

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/dotnet/winforms/pull/10561)

### CODE REVIEW

**Comment by Tanya-Solyanik** [comment]
@Epica3055  - When this PR is done, please prepare this fix for servicing, Thanks!

File: src/System.Windows.Forms/src/System/Windows/Forms/Controls/TabControl/TabPage.cs [hunk 1]
```diff
@@ -502,6 +502,11 @@ internal override void AssignParent(Control? value)
         return (TabPage?)c;
     }
 
+    internal Rectangle GetPageRectangle()
+    {
+        return base.GetToolNativeScreenRectangle();
```
Comment (Tanya-Solyanik) [inline src/System.Windows.Forms/src/System/Windows/Forms/Controls/TabControl/TabPage.cs:507]: Use expression-bodied method please.

File: src/System.Windows.Forms/tests/IntegrationTests/WinformsControlsTest/MainForm.cs [hunk 2]
```diff
@@ -289,7 +293,7 @@ private void UpdateLayout()
 
         ClientSize = new Size(
             (biggestButton.Width + padding * 2) * 3 + padding * 2 + overarchingFlowLayoutPanel.Location.X * 2,
-            (overarchingFlowLayoutPanel.Controls.Count + 1) / 3 * (biggestButton.Height + padding * 2)
+            (int)Math.Ceiling((overarchingFlowLayoutPanel.Controls.Count + 1) / 3.0) * (biggestButton.Height + padding * 2)
```
Comment (Tanya-Solyanik) [inline src/System.Windows.Forms/tests/IntegrationTests/WinformsControlsTest/MainForm.cs:296]: Oops! I missed this!

File: src/System.Windows.Forms/tests/IntegrationTests/WinformsControlsTest/ScrollableControls.cs [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+        public ScrollableControls()
+        {
+            InitializeComponent();
+            this.treeView1.ExpandAll();
+        }
+    }
+}
+
+
+
```
Comment (Tanya-Solyanik) [inline src/System.Windows.Forms/tests/IntegrationTests/WinformsControlsTest/ScrollableControls.cs:17]: Remove the extra blank lines

**Review by Tanya-Solyanik — APPROVED** [review]
Thank you, I added very minor comments, fix them and merge.

**Review by Tanya-Solyanik — DISMISSED** [review]

**Review by SimonZhao888 — APPROVED** [review]

**Pull request merged**